addons/binding/org.openhab.binding.helios/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,28 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Helios Binding
+Bundle-SymbolicName: org.openhab.binding.helios;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .,
+ lib/cxf-core-3.1.0-SNAPSHOT.jar,
+ lib/cxf-services-wsn-api-3.1.0-SNAPSHOT.jar,
+ lib/cxf-services-wsn-core-3.1.0-SNAPSHOT.jar
+Import-Package: com.google.common.collect,
+ com.google.gson,
+ javax.ws.rs.client,
+ javax.ws.rs.core,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.scheduler,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.type,
+ org.eclipse.smarthome.core.types,
+ org.slf4j,
+ org.eclipse.jdt.annotation;resolution:=optional
+Service-Component: OSGI-INF/*.xml
+Export-Package: org.openhab.binding.helios,
```

Comment:
You should also import all packages which you export, see [B.8](http://docs.openhab.org/developers/development/guidelines.html#b-osgi-bundles)